scripts/serve_api.py

```diff
@@ -0,0 +1,71 @@
+"""Serve SourceCred API, for consumption by Observable notebooks.
+
+Usage:
+
+    python3 ./scripts/serve_api.py [PORT]
+
+from the SourceCred repository root. Omit the port to try a default, or
+specify port 0 to pick a free port automatically.
+
+This server binds to `localhost`, which is a loopback address only
+accessible from your machine, not the local network.
+
+Corresponding Observable code:
+
+    sourcecred = {
+      const server = "http://localhost:7007";
+      const blob = await fetch(server).then((r) => r.blob());
+      const esModule = await require(URL.createObjectURL(blob));
+      return esModule.default;
+    }
+
+    myGraph = new sourcecred.core.graph.Graph()  // e.g.
+
+This server serves the same response for all requests.
+
+For best results, run concurrently with `yarn api --watch`. After
+changing SourceCred code, just re-execute the definition of the
+`sourcecred` module in the Observable notebook to live-load the new code
+and update your notebook.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import sys
+from wsgiref import simple_server
+
+
+_API_FILE = os.path.join("dist", "api.js")
+_DEFAULT_PORT = 7007
```

Comment:
nit: mind changing the default port? maybe 9009 or 7008?

I already use 7007 a lot.

Reply:
Sure, will go with 9009. (I picked 7007 because it’s what you used when
running my first prototype of this server, but of course fine to amend!)